docs/content/latest/secure/column-level-encryption.md

```diff
@@ -246,19 +212,21 @@ yugabyte=# select ename, account_number from employees limit 1;
 (1 row)
 ```
 
-### Step 5. Query using `pgp_pub_decrypt`
+### Step 5. Query using pgp_pub_decrypt
 
-Use `pgp_pub_decrypt` and private key for decrypting column data. In order to retrieve the encrypted column data, use `pgp_pub_decrypt` function to decrypt the data and wrap the pgp private key with dearmor function to convert the private key into PGP ASCII-armor format. The Decryption function needs to be used in both SELECT and WHERE clause depending on the query. 
+Use `pgp_pub_decrypt` and private key for decrypting column data. In order to retrieve the encrypted column data, use `pgp_pub_decrypt` function to decrypt the data and wrap the pgp private key with the `dearmor` function to convert the private key into PGP ASCII-armor format. The Decryption function needs to be used in both SELECT and WHERE clause depending on the query.
 
-To allow the decryption, the field name is also casted to the binary data type with the syntax: ` account_number:bytea`.
+To allow the decryption, the field name is also cast to the binary data type with the syntax: `account_number::bytea`.
```

Comment:
You also need to cast the field name to the binary data type by using the syntax `account_number::bytea`.